core/domain/exp_services.py

```diff
@@ -277,17 +279,9 @@ def get_multiple_explorations_by_version(exp_id, version_numbers):
     explorations = []
     exploration_models = exp_models.ExplorationModel.get_multi_versions(
         exp_id, version_numbers)
-    error_versions = []
-    for index, exploration_model in enumerate(exploration_models):
-        try:
-            explorations.append(get_exploration_from_model(exploration_model))
-        except utils.ExplorationConversionError:
-            error_versions.append(version_numbers[index])
-
-    if error_versions:
-        raise Exception(
-            'Exploration %s, versions [%s] could not be converted to latest'
-            'schema version.' % (exp_id, ', '.join(map(str, error_versions))))
+    for _, exploration_model in enumerate(exploration_models):
+        explorations.append(get_exploration_from_model(exploration_model))
```

Comment:
I don't know the reason for this change, can you please explain?

Reply:
Explained above